sdk/xmake.lua

```diff
@@ -188,6 +201,13 @@ target("cherimcu.allocator")
 		target:set('cherimcu.debug-name', "allocator")
 	end)
 
+target("cheriot.object_fast_paths")
+	add_rules("cherimcu.privileged-library", "cherimcu.component-debug")
+	add_files(path.join(coredir, "object_fast_paths/token_unseal.S"))
+	on_load(function (target)
+		target:set("cherimcu.compartment", "object_fast_paths")
```

Comment:
Is this needed?  It's used to override the compartment flag value, but we shouldn't be passing that at all when building a library.